CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Hydrophone is the module responsible for sending emails.
 This API sends notifications to users for things like forgotten passwords, initial signup, and invitations.
 
+## 1.15.0-rc1 - 2023-05-24
```

Comment:
Bump a version

Reply:
What do you mean? I bumped the minor, should I bump the major? :thinking:

Reply:
Don't bump RC when merging

Reply:
Yeah, I won't keep the RC when merging don't worry :slightly_smiling_face: